src/libraries/pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -1,78 +0,0 @@
-﻿<Project DefaultTargets="Build">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
-
-  <PropertyGroup>
-    <TargetFrameworkName>netcoreapp</TargetFrameworkName>
-    <TargetFrameworkVersion>$(NETCoreAppCurrentVersion)</TargetFrameworkVersion>
-    <TargetFramework>$(TargetFrameworkName)$(TargetFrameworkVersion)</TargetFramework>
-    <RefBinDir>$(NETCoreAppPackageRefPath)</RefBinDir>
-    <LibBinDir>$(NETCoreAppPackageRuntimePath)</LibBinDir>
-
-    <IsFrameworkPackage>true</IsFrameworkPackage>
-
-    <!-- Private packages need symbols -->
-    <IncludeSymbolsInPackage Condition="'$(IncludeSymbolsInPackage)' == ''">true</IncludeSymbolsInPackage>
-  </PropertyGroup>
-
-  <ItemGroup>
```

Comment:
@dagood do we have closure validation and type dup checking in final packages?

Reply:
We should have it, but I'm not able to find evidence of it in an official build binlog... targets appear to noop and I don't see why at the moment. Filed https://github.com/dotnet/runtime/issues/31805.